Resources/Locale/en-US/deltav/objectives/conditions/teach-person.ftl

```diff
@@ -0,0 +1 @@
+﻿objective-condition-teach-person-title = Teach {$targetName}, {CAPITALIZE($job)} a lesson
```

Comment:
```suggestion
﻿objective-condition-teach-person-title = Teach {$targetName}, {CAPITALIZE($job)} a lesson. You need to be within 30 meters of the target, otherwise the syndicate can't take credit for the job.
```